app/hotels/src/navigation/singleHotel/SingleHotelNavigationScreen.js

```diff
@@ -61,18 +62,17 @@ class SingleHotelNavigationScreen extends React.Component<Props> {
 
   render() {
     return (
-      <SingleHotel
-        onGoToPayment={this.goToPayment}
-        onGoToMap={this.goToMap}
-        currency={this.props.currency}
-        language={this.props.language}
```

Comment:
Is `language` not necessary at all anymore?

Reply:
When we started with hotels, we had this prop passed in from native. Now we can fetch it from `DeviceInfo`

Reply:
I see!